packages/ssx-sdk/src/modules/Storage/KeplerStorage.ts

```diff
@@ -175,8 +175,12 @@ export class KeplerStorage implements IStorage, IKepler {
     return this.orbit.delete(`${prefix}/${key}`, request);
   }
 
-  public async deleteAll(prefix: string = this.prefix): Promise<Response[]> {
-    return this.orbit.deleteAll(prefix);
+  public async deleteAll(prefix?: string): Promise<Response[]> {
+    if (!!prefix) {
+      return this.orbit.deleteAll(`${this.prefix}${prefix}`);
```

Comment:
```suggestion
      return this.orbit.deleteAll(`${this.prefix}/${prefix}`);
```

I guess this line should have a `/` as we have on list or delete methods.